### ISSUE TITLE
Build fails since the latest release

### ISSUE THREAD

**Issue by majamee** [mutschler/mt#10]
Since latest release 1.0.9 the build instructions fail and I cannot find a precompiled package. Here the terminal output of the failed build attempt:

`[majamee@gh0st mt]$ go get github.com/mutschler/mt`
`# github.com/mutschler/mt/vendor/gitlab.com/opennota/screengen
cgo-gcc-prolog: In Funktion »_cgo_eca694cc866a_Cfunc_av_register_all«:
cgo-gcc-prolog:131:2: Warnung: »av_register_all« ist veraltet [-Wdeprecated-declarations]
In file included from ../../go/src/github.com/mutschler/mt/vendor/gitlab.com/opennota/screengen/screengen.go:20:
/usr/include/libavformat/avformat.h:2043:6: Anmerkung: hier deklariert
 void av_register_all(void);
      ^~~~~~~~~~~~~~~`
`[majamee@gh0st mt]$ go install github.com/mutschler/mt`
`# github.com/mutschler/mt/vendor/gitlab.com/opennota/screengen
cgo-gcc-prolog: In Funktion »_cgo_eca694cc866a_Cfunc_av_register_all«:
cgo-gcc-prolog:131:2: Warnung: »av_register_all« ist veraltet [-Wdeprecated-declarations]
In file included from ../../go/src/github.com/mutschler/mt/vendor/gitlab.com/opennota/screengen/screengen.go:20:
/usr/include/libavformat/avformat.h:2043:6: Anmerkung: hier deklariert
 void av_register_all(void);
      ^~~~~~~~~~~~~~~`

**Comment by mutschler** [comment]
Hi, which version of ffmpeg did you use? and which OS are you trying to build mt on?

Automatic builds are currently broken the way they used to work, so i have to think about something different.

**Comment by majamee** [comment]
Hi, please have a look here for all the details and the build fail:
https://hub.docker.com/r/majamee/alpine-dash-hls/builds/bc3rt6z2byeu5lahavjxg9a/

**Comment by majamee** [comment]
I just did, but so far still unsuccessful:
https://hub.docker.com/r/majamee/alpine-dash-hls/builds/bjh2dtjtbbrwybda7beyarb/

Thank you for looking into it though.

**Comment by mutschler** [comment]
Ok so I've just tried everything with the latest alpine image from docker and it seems like the main Problem with your setup is that you are cloning the repo outside of your GOPATH when using your file 1:1 I've got stuck as well:

```
git clone https://github.com/mutschler/mt.git /tmp/mt && \
cd /tmp/mt && go get github.com/mutschler/mt && \
go build && mv mt /bin/mt && rm -rf /tmp && \
```

go is pretty strict about where you can put code to compile it and where not, so you'll have two options
1. just use go get on mt which should then put it to `~/go/src/github.com/mutschler/mt/` and also drop a binary in `~/go/bin/` afterwards for you to use/copy
2. you can use a different `GOPATH` if you'd like to but I don't see the need for that

so basically running something like this instead of the 3 lines above should work:

```
go get github.com/mutschler/mt && mv ~/go/bin/mt /bin/mt
```

**Comment by majamee** [comment]
Thank you, I updated my Dockerfile according to your suggestions and now everything seems to be functional again: 
https://hub.docker.com/r/majamee/alpine-dash-hls/builds/bbfs5vetqtgzkuvxm9p4xg8/

**Comment by mutschler** [comment]
Glad to hear that!
